tools/generator/test/games/test.yml

```diff
@@ -198,6 +214,24 @@ function:
     fct_ret_type: struct_with_array array
     fct_arg:
       - [l, struct_with_array array, the list]
+  -
+    fct_name: send_me_tuple_struct
+    fct_summary: Send me this struct with value (42, true)
+    fct_ret_type: void
+    fct_arg:
+      - [s, simple_tuple, the tuple]
+  -
+    fct_name: send_me_struct_with_struct
+    fct_summary: Send me this struct with the usual test values for these fields
+    fct_ret_type: void
+    fct_arg:
+      - [s, struct_with_struct, the struct]
+  -
+    fct_name: send_me_tuple_with_struct
```

Comment:
Right now you never check that `send_me_struct_with_struct` and `send_me_tuple_with_struct` are called.

Reply:
My bad, I forgot. I also need to add arrays into the mix for this PR to be ready